src/models/krate_reverse_dependencies.sql

```diff
@@ -1,42 +1,32 @@
-SELECT
-    dependencies.*, crate_downloads, crate_name, total
-FROM (
-    -- Apply pagination to the crates
-    SELECT *, COUNT(*) OVER () as total FROM (
-        SELECT
-            crate_downloads.downloads AS crate_downloads,
-            crates.name AS crate_name,
-            versions.id AS version_id
-        FROM
-        -- We only want the crates whose *max* version is dependent, so we join on a
-        -- subselect that includes the versions with their ordinal position
-        (
-            SELECT DISTINCT ON (crate_id)
-               crate_id, semver_no_prerelease, id
+WITH default_versions as (
```

Comment:
maybe use a different CTE name again here to avoid confusion with the database table :)

Reply:
No problem!